compiler/src/main/scala/play/twirl/compiler/TwirlCompiler.scala

```diff
@@ -162,8 +162,13 @@ case class GeneratedSourceVirtual(path: String) extends AbstractGeneratedSource
 
 object TwirlCompiler {
 
+  var isScala3 = false
+
```

Comment:
I think this `var isScala3` is not necessary.
If you declare `emitScala3Sources` as a val, it can be referred to later as sc.emitScala3Sources.